oop-relationships/oop-relationships.md

```diff
@@ -0,0 +1,132 @@
+# OOP Relationships
```

Comment:
I'm EXTREMELY into this conversation! I love love love love it, I want us (Ada) to keep exploring this direction.

For the sake of giving an explanation for C15: Choosing inheritance and composition as the two things to fixate on during OOP is largely me conforming to what the old Ada curriculum emphasized, and trying to soften this curriculum transition. I'm very excited to slowly evolve this, though.

Reply:
I'm not really calling for a major change in how those two facets are presented. They're both important concepts.  I'm just venting that dynamic typing muddies the explanation soooo much.  It would be great if students could get introduced to some of the restrictions imposed by static typing as part of the official curriculum.  I feel like a lot of students get quite a bit of culture shock if the first time they run into statically-typed languages is on their capstone.